src/pages/Transaction/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Switch, Route, useRouteMatch } from 'react-router-dom';
+
+import MainContainer from 'parts/MainContainer';
+import { URL_PARAMETERS } from 'utils/constants/links';
+
+const IssueTransaction = React.lazy(() => import(/* webpackChunkName: 'issue-transaction' */ './IssueTransaction'));
+const RedeemTransaction = React.lazy(() => import(/* webpackChunkName: 'redeem-transaction' */ './RedeemTransaction'));
+const ReplaceTransaction = React.lazy(
+  () => import(/* webpackChunkName: 'replace-transaction' */ './ReplaceTransaction')
+);
```

Comment:
These nested page components are just for routing and no specific implementation yet.